<?xml version="1.0" encoding="UTF-8"?>
<ui version="4.0">
 <class>productAdder</class>
 <widget class="QWidget" name="productAdder">
  <property name="geometry">
   <rect>
    <x>0</x>
    <y>0</y>
    <width>410</width>
    <height>386</height>
   </rect>
  </property>
  <property name="windowTitle">
   <string>Добавить</string>
  </property>
  <widget class="QLabel" name="label">
   <property name="geometry">
    <rect>
     <x>0</x>
     <y>-20</y>
     <width>411</width>
     <height>101</height>
    </rect>
   </property>
   <property name="text">
    <string>&lt;!DOCTYPE HTML PUBLIC &quot;-//W3C//DTD HTML 4.0//EN&quot; &quot;http://www.w3.org/TR/REC-html40/strict.dtd&quot;&gt;
&lt;html&gt;&lt;head&gt;&lt;meta name=&quot;qrichtext&quot; content=&quot;1&quot; /&gt;&lt;style type=&quot;text/css&quot;&gt;
p, li { white-space: pre-wrap; }
&lt;/style&gt;&lt;/head&gt;&lt;body style=&quot; font-family:'MS Shell Dlg 2'; font-size:8pt; font-weight:400; font-style:normal;&quot;&gt;
&lt;p align=&quot;center&quot; style=&quot; margin-top:12px; margin-bottom:12px; margin-left:0px; margin-right:0px; -qt-block-indent:0; text-indent:0px;&quot;&gt;Назовите блюдо и выберите нужные ингредиенты. Вы увидите&lt;br /&gt;себестоимость блюда и сможете установить свою цену. Если&lt;br /&gt;продукта нет в списке - добавьте его во вкладке &amp;quot;Продукты&amp;quot;&lt;/p&gt;&lt;/body&gt;&lt;/html&gt;</string>
   </property>
  </widget>
  <widget class="QLabel" name="label_2">
   <property name="geometry">
    <rect>
     <x>10</x>
     <y>80</y>
     <width>111</width>
     <height>16</height>
    </rect>
   </property>
   <property name="text">
    <string>Наименование:</string>
   </property>
  </widget>
  <widget class="QLabel" name="proteinLine">
   <property name="geometry">
    <rect>
     <x>10</x>
     <y>100</y>
     <width>90</width>
     <height>16</height>
    </rect>
   </property>
   <property name="text">
    <string>Цена (руб):</string>
   </property>
  </widget>
  <widget class="QLineEdit" name="nameAdder">
   <property name="geometry">
    <rect>
     <x>130</x>
     <y>80</y>
     <width>113</width>
     <height>20</height>
    </rect>
   </property>
  </widget>
  <widget class="QLineEdit" name="priceAdder">
   <property name="geometry">
    <rect>
     <x>130</x>
     <y>100</y>
     <width>113</width>
     <height>20</height>
    </rect>
   </property>
  </widget>
  <widget class="Line" name="line">
   <property name="geometry">
    <rect>
     <x>10</x>
     <y>92</y>
     <width>121</width>
     <height>16</height>
    </rect>
   </property>
   <property name="orientation">
    <enum>Qt::Horizontal</enum>
   </property>
  </widget>
  <widget class="Line" name="line_2">
   <property name="geometry">
    <rect>
     <x>10</x>
     <y>112</y>
     <width>121</width>
     <height>16</height>
    </rect>
   </property>
   <property name="orientation">
    <enum>Qt::Horizontal</enum>
   </property>
  </widget>
  <widget class="QPushButton" name="addConfirmButton">
   <property name="geometry">
    <rect>
     <x>314</x>
     <y>360</y>
     <width>91</width>
     <height>23</height>
    </rect>
   </property>
   <property name="text">
    <string>Добавить</string>
   </property>
  </widget>
  <widget class="QPushButton" name="cancelAdderButton">
   <property name="geometry">
    <rect>
     <x>240</x>
     <y>360</y>
     <width>75</width>
     <height>23</height>
    </rect>
   </property>
   <property name="text">
    <string>Отмена</string>
   </property>
  </widget>
  <widget class="QLabel" name="adderStatusText">
   <property name="geometry">
    <rect>
     <x>10</x>
     <y>60</y>
     <width>391</width>
     <height>20</height>
    </rect>
   </property>
   <property name="text">
    <string/>
   </property>
   <property name="alignment">
    <set>Qt::AlignCenter</set>
   </property>
  </widget>
  <widget class="QTableWidget" name="ingredientsToAdd">
   <property name="geometry">
    <rect>
     <x>10</x>
     <y>130</y>
     <width>231</width>
     <height>221</height>
    </rect>
   </property>
   <column>
    <property name="text">
     <string>Наименование</string>
    </property>
   </column>
   <column>
    <property name="text">
     <string>Масса (в граммах)</string>
    </property>
   </column>
  </widget>
  <widget class="QListWidget" name="ingredientsShow">
   <property name="geometry">
    <rect>
     <x>250</x>
     <y>80</y>
     <width>151</width>
     <height>271</height>
    </rect>
   </property>
  </widget>
  <widget class="QLabel" name="selfPrice">
   <property name="geometry">
    <rect>
     <x>10</x>
     <y>370</y>
     <width>231</width>
     <height>16</height>
    </rect>
   </property>
   <property name="text">
    <string>Себестоимость:</string>
   </property>
  </widget>
  <widget class="QPushButton" name="deleteIngredient">
   <property name="enabled">
    <bool>false</bool>
   </property>
   <property name="geometry">
    <rect>
     <x>10</x>
     <y>350</y>
     <width>161</width>
     <height>23</height>
    </rect>
   </property>
   <property name="text">
    <string>Удалить ингредиент</string>
   </property>
  </widget>
 </widget>
 <resources/>
 <connections/>
</ui>
